mkosi/config.py

```diff
@@ -608,11 +608,11 @@ def output_nspawn_settings(self) -> Path:
 
     @property
     def output_checksum(self) -> Path:
-        return Path("SHA256SUMS")
+        return with_output_dir(self, "SHA256SUMS")
```

Comment:
How about we use `build_auxiliary_output_path()` for these as well? Otherwise we're going to run into issues when using the same output directory for multiple images

Reply:
Happy to do that, but that would then generate files like `MyImage_123.SHA256SUMS` or something close to it. If that is OK then let's do it. I'll need to adjust the check for `SHA256SUMS` too.

Reply:
Yeah I think its fine to generate files like that. Let's go with this approach, it keeps things consistent with the naming of the other output files